Assets/Scripts/StandardSamples/Discord/DiscordManager.cs

```diff
@@ -121,13 +123,54 @@ private class TokenReader
         {
             public string access_token;
             public float expires_in;
+            public string refresh_token;
         }
 
         private string cachedToken = null;
+        private string refresh_token = null;
```

Comment:
This should be `camelCase` instead of `snake_case`.

Reply:
This is resolved as you suggested, changing the name of this to refreshToken.